skynet/modules/ttt/summaries/processor.py

```diff
@@ -33,9 +33,11 @@ def initialize():
     global llm
 
     llm = ChatOpenAI(
+        model=llama_path,
         api_key='placeholder',  # use a placeholder value to bypass validation, and allow the custom base url to be used
         base_url=f'{openai_api_base_url}/v1',
         default_headers={"X-Skynet-UUID": app_uuid},
+        frequency_penalty=1,
```

Comment:
What's this for?

Reply:
https://platform.openai.com/docs/advanced-usage/parameter-details

I used this to remove cases where the llm started hallucinating and repeating the same stuff over and over until it filled the context window

Reply:
Nice!